draft-ietf-tls-tls13.md

```diff
@@ -2785,6 +2792,22 @@ certificate_authorities
   the certificate_authorities list is empty, then the client MAY
   send any certificate of the appropriate ClientCertificateType,
   unless there is some external arrangement to the contrary.
+
+certificate_extensions
+: A list of certificate extension OIDs [RFC5280] with their allowed
+  values, represented in DER-encoded format. Some certificate
+  extension OIDs allow multiple values (e.g. Extended Key Usage).
```

Comment:
An OID with multiple values.

From: ekr [mailto:notifications@github.com]
Sent: Tuesday, August 18, 2015 7:56 AM
To: tlswg/tls13-spec <tls13-spec@noreply.github.com>
Cc: Andrei Popov <Andrei.Popov@microsoft.com>
Subject: Re: [tls13-spec] TLS Client Auth (#209)


In draft-ietf-tls-tls13.md<https://github.com/tlswg/tls13-spec/pull/209#discussion_r37308043>:

> @@ -2785,6 +2792,22 @@ certificate_authorities

>    the certificate_authorities list is empty, then the client MAY

>    send any certificate of the appropriate ClientCertificateType,

>    unless there is some external arrangement to the contrary.

> +

> +certificate_extensions

> +: A list of certificate extension OIDs [RFC5280] with their allowed

> +  values, represented in DER-encoded format. Some certificate

> +  extension OIDs allow multiple values (e.g. Extended Key Usage).

Does that mean that I can supply two instances of an OID or an OID with two values?

—
Reply to this email directly or view it on GitHub<https://github.com/tlswg/tls13-spec/pull/209/files#r37308043>.